client/my-sites/checkout/composite-checkout/components/wp-checkout-order-review.js

```diff
@@ -48,8 +52,15 @@ export default function WPCheckoutOrderReview( {
 	const hasRenewal = Boolean(
 		items.find( ( { wpcom_meta } ) => 'renewal' === wpcom_meta?.extra?.purchaseType )
 	);
+	const currentPlanProductSlug = useSelector( ( state ) => getCurrentPlan( state, siteId ) )
```

Comment:
Minor: since `siteId` can, I think, be `undefined`, we can save a little bit of work here by making this: 

```js
	const currentPlanProductSlug = useSelector( ( state ) => siteId && getCurrentPlan( state, siteId ) )?.productSlug;
```

I've run into this many times recently when converting components to TypeScript.